skimage/filters/thresholding.py

```diff
@@ -425,52 +432,61 @@ def threshold_minimum(image, nbins=256, bias='min'):
     """
 
     def find_local_maxima(hist):
+        # We can't use scipy.signal.argrelmax
+        # as it fails on plateaus
         maximums = list()
-        direction = +1
+        direction = 1
         for i in range(hist.shape[0] - 1):
             if direction > 0:
                 if hist[i + 1] < hist[i]:
                     direction = -1
                     maximums.append(i)
             else:
                 if hist[i + 1] > hist[i]:
-                    direction = +1
+                    direction = 1
         return maximums
 
+    if bias not in ('min', 'mid', 'max'):
+        raise ValueError("Unknown bias: {0}".format(bias))
+
     hist, bin_centers = histogram(image.ravel(), nbins)
     # On blank images (e.g. filled with 0) with int dtype, `histogram()`
     # returns `bin_centers` containing only one value. Speed up with it.
     if bin_centers.size == 1:
         return bin_centers[0]
 
     threshold = 0
+    maximums = 0
```

Comment:
> hum, maximums has to be defined, no?

Of course. I was thinking about initializing `maximums` to `None` and changing the loop condition to take into account the initial value. There could be many ways to implement this loop.